apple/RNGestureHandler.mm

```diff
@@ -471,6 +471,15 @@ - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer
       }
     }
   }
+
+  if (handler != nil) {
+    for (NSNumber *handlerTag in handler->_handlersThatShouldWait) {
+      if ([_tag isEqual:handlerTag]) {
+        return YES;
+      }
```

Comment:
Shouldn't we also check `[_handlersThatShouldWait count]`, as above?

Reply:
I moved the null check up and removed the check for `[_handlersThatShouldWait count]` entirely - I don't really see the difference it makes when the list is empty.